packages/plugin-kit/src/createSliceMachineHelpers.ts

```diff
@@ -74,6 +74,28 @@ export class SliceMachineHelpers {
 		};
 	};
 
+	updateProjectConfig = async (
+		config: SliceMachineConfig,
+		format?: boolean,
```

Comment:
Rather than using a positional argument, which doesn't scale to more arguments, we could use named arguments as the second argument:

```ts
type UpdateProjectConfigConfig = {
  format?: boolean
}

// ...

updateProjectConfig(
  sliceMachineConfig: SliceMachineConfig,
  config: UpdateProjectConfigConfig = {}
) {
  // ...
}
```